micromamba.el

```diff
@@ -48,11 +48,27 @@
   "Micromamba (environment manager) integration for Emacs."
   :group 'python)
 
+(defcustom micromamba-home (expand-file-name "~/.micromamba") ;; adapted from conda.el
+  "The directory where micromamba stores its files."
+  :type 'directory
+  :group 'micromamba)
+
 (defcustom micromamba-executable (executable-find "micromamba")
   "Path to micromamba executable."
   :type 'string
   :group 'micromamba)
 
+(defcustom micromamba-message-on-environment-switch t ;;adapted from conda.el
+  "Whether to message when switching environments.  Default true."
+  :type 'boolean
+  :group 'micromamba)
+
+(defcustom micromamba-activate-base-by-default nil ;;adapted from conda.el
+  "Whether to activate the base environment by default if no other is preferred.
+Default nil."
+  :type 'boolean
+  :group 'micromamba)
```

Comment:
`micromamba` doesn't have `base`, so this doesn't make much sense. I suggest making a variable like `micromamba-fallback-environment`, i.e. an environment that `micromamba-env-autoactivate-mode` activates by default.

With that in mind, parsing `micromamba config list --json` to check for `auto_activate_base` also doesn't seem necessary.